src/server/campaignd/server.cpp

```diff
@@ -433,7 +433,7 @@ void server::load_config()
 			config data;
 			in = filesystem::istream_file(filesystem::normalize_path(addon_file));
 			read_gz(data, *in);
-			if(!data) {
+			if ((false)) {
```

Comment:
I think this should probably look something like `if(!read_gz(data, *in))`. I don't know exactly what `read_gz` looks like so it might be a bit different, but something in that spirit.